CMakeLists.txt

```
@@ -135,7 +135,10 @@ set(WITH_TARGET_AARCH64 no CACHE BOOL "Enable target AARCH64")
 set(WITH_TARGET_X86 no CACHE BOOL "Enable target X86")
 set(WITH_TARGET_WASM no CACHE BOOL "Enable target WebAssembly")
 if (WITH_LLVM)
-    find_package(StaticZSTD REQUIRED)
+    set(WITH_ZSTD yes CACHE BOOL "Detect (and required static libzstd for llvm")
```

Did you mean to write this:
```suggestion
    set(WITH_ZSTD yes CACHE BOOL "Detect (and require) static libzstd for llvm")
```